pkgs/data/fonts/sarabun/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "sarabun";
+  version = "unstable-2018-08-24";
```

Comment:
As far as I am aware, we're using the date the package is put into nixpkgs rather than the date of the last commit to the github repo... but I might be wrong here :thinking:

Reply:
That wouldn’t be a useful metric. You want to be able to check the date of the latest commit in the upstream project to see if there’s a later version. All packages I’ve maintained have used the upstream’s date.

Reply:
Okay :shrug: